src/server/game/Entities/Player/Player.cpp

```diff
@@ -15973,12 +15973,7 @@ void Player::AreaExploredOrEventHappens(uint32 questId)
             QuestStatusData& q_status = m_QuestStatus[questId];
 
             if (!q_status.Explored)
-            {
                 q_status.Explored = true;
-                m_QuestStatusSave[questId] = QUEST_DEFAULT_SAVE_TYPE;
```

Comment:
Because it won't save the completed objective to DB without it, right?

Sorry, didn't think about that.